src/collections/_documentation/platforms/javascript/angular.md

```diff
@@ -47,6 +105,48 @@ export class SentryErrorHandler implements ErrorHandler {
 export class AppModule {}
 ```
 
+When using your own `ErrorHandler`, make sure that whenever you use `HttpInterceptor` alongside it,
+it doesn't modify the error captured originally.
```

Comment:
> it doesn't modify the error captured originally.

I'm not 100% clear here if "it" is the custom error handler or the http interceptor. Can we specify which one?

Reply:
Updated